src/ngraph_builder.cc

```diff
@@ -1597,7 +1665,9 @@ static Status TranslateDepthwiseConv2dNativeOp(
                                           input_shape[3]};
     auto ng_sliced_input =
         make_shared<ng::op::Slice>(ng_input, lower_bound, upper_bound);
-
+    if (!is_nhwc) {
+      SaveNgOp(first_ng_op_map, op->name(), ng_input);
```

Comment:
Hi @shresthamalik, this line 1669  `SaveNgOp(first_ng_op_map, op->name(), ng_input);` seems collide with line 1635  `SaveNgOp(first_ng_op_map, op->name(), ng_input);`, so no matter is_nhwc or not, we are saving this ng_input op.
Wondering here is this `ng_input` to be `ng_sliced_input`?

Reply:
same question as above here.

Reply:
should be ng_sliced_input. Thanks!